src/Modules/ModParser.lua

```diff
@@ -3377,6 +3384,7 @@ local specialModList = {
 	["recover energy shield equal to (%d+)%% of armour when you block"] = function(num) return { mod("EnergyShieldOnBlock", "BASE", 1,  { type = "PercentStat", stat = "Armour", percent = num }) } end,
 	["(%d+)%% of damage taken while affected by clarity recouped as mana"] = function(num) return { mod("ManaRecoup", "BASE", num, { type = "Condition", var = "AffectedByClarity" }) } end,
 	["recoup effects instead occur over 3 seconds"] = { flag("3SecondRecoup") },
+	["life recoup effects instead occur over 3 seconds"] = { flag("3SecondRecoup") },
```

Comment:
This should only apply to life, not to all

Reply:
Of course, not sure how I missed that. Fixing!